strawberry/cli/commands/export_schema.py

```diff
@@ -0,0 +1,13 @@
+import importlib
+
+import click
+
+from strawberry.printer import print_schema
+
+
+@click.command("export_schema", short_help="Exports the schema")
```

Comment:
https://click.palletsprojects.com/en/7.x/api/#click.command
> The name of the command defaults to the name of the function with underscores replaced by dashes. If you want to change that, you can pass the intended name as the first argument.

We should use the default behavior here and allow `click` to pick the command name. Dashes are standard in CLIs

Reply:
Good to know!

(I guess there is also no need for the existing `server` command to specify its name explicitly, right?)

Reply:
Probably not 🙂